.circleci/config.yml

```diff
@@ -196,7 +196,7 @@ jobs:
           command: |
             source activate venv
             cd /tmp/data/five-echo/
-            tedana -d p06.SBJ01_S09_Task11_e[1,2,3,4,5].sm.nii.gz -e 15.4 29.7 44.0 58.3 72.6 --verbose
+            tedana -d p06.SBJ01_S09_Task11_e[1,2,3,4,5].sm.nii.gz -e 15.4 29.7 44.0 58.3 72.6 --verbose --sourceTEs 0
```

Comment:
The optimal combination is probably what we should be using, here !

Reply:
I'll revert it now.